packages/sdk/contractkit/src/wrappers/Election.test.ts

```diff
@@ -0,0 +1,257 @@
+import { CeloTxReceipt } from '@celo/connect/lib/types'
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { addressToPublicKey } from '@celo/utils/lib/signatureUtils'
+import BigNumber from 'bignumber.js'
+import Web3 from 'web3'
+import { mineToNextEpoch } from '../test-utils/utils'
+
+import { zeroAddress } from 'ethereumjs-util'
+import { newKitFromWeb3 } from '../kit'
+import { AccountsWrapper } from './Accounts'
+import { ElectionWrapper } from './Election'
+import { LockedGoldWrapper } from './LockedGold'
+import { ValidatorsWrapper } from './Validators'
+
+const minLockedGoldValue = Web3.utils.toWei('10000', 'ether') // 10k gold
+
+const blsPublicKey =
+  '0x4fa3f67fc913878b068d1fa1cdddc54913d3bf988dbe5a36a20fa888f20d4894c408a6773f3d7bde11154f2a3076b700d345a42fd25a0e5e83f4db5586ac7979ac2053cd95d8f2efd3e959571ceccaa743e02cf4be3f5d7aaddb0b06fc9aff00'
+const blsPoP =
+  '0xcdb77255037eb68897cd487fdd85388cbda448f617f874449d4b11588b0b7ad8ddc20d9bb450b513bb35664ea3923900'
+
+testWithGanache('Election Wrapper', (web3) => {
+  const ZERO_GOLD = new BigNumber(web3.utils.toWei('0', 'ether'))
+  const ONE_HUNDRED_GOLD = new BigNumber(web3.utils.toWei('100', 'ether'))
+  const ONE_HUNDRED_ONE_GOLD = new BigNumber(web3.utils.toWei('101', 'ether'))
+  const TWO_HUNDRED_GOLD = new BigNumber(web3.utils.toWei('200', 'ether'))
+  const TWO_HUNDRED_ONE_GOLD = new BigNumber(web3.utils.toWei('201', 'ether'))
+  const THREE_HUNDRED_GOLD = new BigNumber(web3.utils.toWei('300', 'ether'))
+
+  const kit = newKitFromWeb3(web3)
+  let accounts: string[] = []
+  let election: ElectionWrapper
+  let accountsInstance: AccountsWrapper
+  let validators: ValidatorsWrapper
+  let lockedGold: LockedGoldWrapper
+
+  beforeAll(async () => {
+    accounts = await kit.connection.getAccounts()
+
+    election = await kit.contracts.getElection()
+
+    validators = await kit.contracts.getValidators()
+
+    lockedGold = await kit.contracts.getLockedGold()
+
+    accountsInstance = await kit.contracts.getAccounts()
+  })
+
+  afterAll(async () => {
+    kit.connection.stop()
+  })
+
+  const registerAccountWithLockedGold = async (
+    account: string,
+    value: string = minLockedGoldValue
+  ) => {
+    if (!(await accountsInstance.isAccount(account))) {
+      await accountsInstance.createAccount().sendAndWaitForReceipt({ from: account })
+    }
+    await lockedGold.lock().sendAndWaitForReceipt({ from: account, value })
+  }
+
+  const setupGroup = async (groupAccount: string, members: number = 1) => {
```

Comment:
Is the default useful here, or would it be better to always explicitly specify?

Reply:
removed `members`, since the number of group members is not too relevant for the election contract, and is unlikely to be used in this test.